pennylane/transforms/__init__.py

```diff
@@ -20,38 +20,32 @@
 Transforms
 ----------
 
-Transforms that act on QNodes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+How to build a custom PennyLane transform?
+------------------------------------------
```

Comment:
There is an empty heading here: 
![image](https://github.com/PennyLaneAI/pennylane/assets/2959003/510a2dfc-69a3-467f-9935-4fca9fa4e954)

I suggest merging these two headings into a single one:

```rest
Custom transforms
-----------------
```